src/index.js

```diff
@@ -1,59 +1,41 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import * as serviceWorker from './serviceWorker';
-import ApolloClient from 'apollo-boost';
-import { gql } from "apollo-boost";
 import { ApolloProvider } from '@apollo/react-hooks';
 import { BrowserRouter, Route } from 'react-router-dom';
+import { createBrowserHistory } from "history";
 
 import 'bootstrap/dist/css/bootstrap.min.css';
-import SignUp from './signupPage';
-import SignIn from './signInPage';
+import SignUp from './pages/signupPage'; //Si hay una carpeta ya de una toma el idex.js de esa carpeta
+import SignIn from './pages/signInPage';
+import Home from './pages/home';
+import NavbarComp from "./components/navbar.js"
 
-const client = new ApolloClient({
-  uri: 'http://localhost:3001/graphql',
-});
+import client from './apollo/configurations/client' 
 
+const history = createBrowserHistory();
 
-
-/*
-client
-  .query({
-    query: gql`
-      {
-        user(id: "2c5ea4c0-4067-11e9-8bad-9b1deb4d3b7d"){
-          firstName
-          lastName
-          username
-        }
-      }
-    `
-  })
-  .then(result => console.log(result)); // No me esta devolviendo el result, pero la query la hace 
-
-console.log(client)
-*/
+const view = (Page) => (
+  <div><NavbarComp></NavbarComp>{Page}</div>
+);
 
 const App = () => (
   <ApolloProvider client={client}>
     <BrowserRouter>
-      <Route exact={true} path="/" render={() => <SignUp></SignUp>}></Route>
-      <Route exact={true} path="/signup" render={() => <SignUp></SignUp>}></Route>
-      <Route exact={true} path="/signin" render={() => <SignIn></SignIn>}></Route>
+      <Route exact={true} history={history} path="/" render={() => view(<SignIn></SignIn>)}></Route>
```

Comment:
You can write this shorter:

```suggestion
      <Route exact path="/" render={() => view(<SignIn/>)} />
```

Also, you don't need to pass `history` anymore.